crawl-ref/source/evoke.cc

```diff
@@ -520,13 +522,58 @@ bool skill_has_manual(skill_type skill)
     return manual_slot_for_skill(skill) != -1;
 }
 
+void archaeologist_open_crate(item_def& crate)
+{
+    unrand_type type = (unrand_type)crate.props[ARCHAEOLOGIST_CRATE_ITEM].get_int();
+    make_item_unrandart(crate, type);
+    item_colour(crate);
+    item_set_appearance(crate);
+    mprf("The crate's locking mechanism finally gives in... Revealing %s!", crate.name(DESC_THE).c_str());
+    crate.props.erase(ARCHAEOLOGIST_CRATE_ITEM);
+}
+
+void archaeologist_read_tome(item_def& tome)
+{
+    tome.base_type = OBJ_BOOKS;
+    tome.sub_type = BOOK_MANUAL;
+    tome.skill_points = 1000;
```

Comment:
This should probably be 700. As it stands, I feel it unlocks a bit too late and it might be tempting to skip weaker unrands altogether and go for something else.

Reply:
I'll do some maths on when this unlocks normally and see what it ends up at. Also, it will unlock earlier for low-experience-apt races I believe, which might be something I have to look into.

Reply:
Yes, that sounds about right :) before Lair is the target I had in mind.